core/server/models/session.js

```diff
@@ -0,0 +1,54 @@
+const ghostBookshelf = require('./base');
+
+let Session,
+    Sessions;
+
+Session = ghostBookshelf.Model.extend({
+    tableName: 'sessions',
+
+    parse: function parse(attrs) {
+        if (!attrs.session_data) {
```

Comment:
What is the case? `session_data` is not nullable, right?

Reply:
good point - need to double check there isn't a case for this though